zoloto/viewer.py

```diff
@@ -23,32 +32,32 @@ def __init__(self, camera: BaseCamera, *, title="Camera Viewer", annotate=False)
 
         signal.signal(signal.SIGINT, self.signal_handler)
 
-    def show_frame(self):
+    def show_frame(self) -> None:
         frame = self.camera.capture_frame()
         if self.annotate:
             self.camera._annotate_frame(frame)
         frame = self.on_frame(frame)
         tkinter_image = PhotoImage(
             image=Image.fromarray(cv2.cvtColor(frame, cv2.COLOR_BGR2RGBA))
         )
-        self.label.imagetk = tkinter_image
+        self.label.imagetk = tkinter_image  # type: ignore
         self.label.configure(image=tkinter_image)
         self.label.after(10, self.show_frame)
 
-    def signal_handler(self, sig, frame):
+    def signal_handler(self, sig: signal.Signals, frame: FrameType) -> None:
         self.stop()
 
-    def on_frame(frame):
+    def on_frame(self, frame: ndarray) -> ndarray:
         return frame
 
-    def start(self):
+    def start(self) -> None:
         self.show_frame()
         self.window.mainloop()
 
-    def stop(self):
-        self.window.quit()
+    def stop(self) -> None:
+        self.window.quit()  # type: ignore
```

Comment:
Why's the type being ignored here?

Reply:
The type stubs for `Tkinter` don't include this function.

`zoloto/viewer.py:58: error: Call to untyped function "quit" in typed context`

Reply:
waaaat

ok, I can live with that for now.